Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkController.cs

```diff
@@ -109,6 +110,39 @@ ThemeColorController FindThemeColorController()
             return controllerTransform.GetComponent<ThemeColorController>();
         }
 
+        private void SetTransformSync(Transform transf, int syncMode, int targetMode)
+        {
+
+            #if UDONSHARP
+
+            transf.GetComponent<UdonBehaviour>().enabled = syncMode < targetMode;
+
+            #endif
+
+        }
+
+        void SetControllerSyncMode(int syncMode)
+        {
+            
+            SetTransformSync(gainSlider.transform, syncMode, 2);
```

Comment:
Why the magic numbers? Are these supposed to match `ControllerSyncMode`? If so, please use the enum type throughout. `<` etc are defined for enums

Reply:
Yes, didn't see that before